test/services/autoVerifyPsychologist.spec.ts

```diff
@@ -1,31 +1,69 @@
 import sinon from 'sinon';
-import verifyPsychologist from '../../services/demarchesSimplifiees/verifyPsychologist';
 import autoVerifyPsychologists from '../../services/demarchesSimplifiees/autoVerify';
+import config from '../../utils/config';
+import graphql from '../../utils/sendGraphQLRequest';
 
 describe('autoVerifyPsychologist', () => {
-  let verifyPsychologistStub;
+  let executeMutationStub;
 
   beforeEach(() => {
-    verifyPsychologistStub = sinon.stub(verifyPsychologist, 'verifyPsychologist');
+    executeMutationStub = sinon.stub(graphql, 'executeMutation');
```

Comment:
With typescript changes, we were testing only the scope of the method. So we should stub `verifyPsychologist` here, and not `executeMutation`. The test will also change since the arguments checked will not be the same as before.